typescript-playwright-sample/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "typescript-playwright-sample",
+  "version": "0.0.0",
+  "description": "Sample project that demonstrates accessibility testing with TypeScript, Playwright @axe-core/playwright, SARIF, and Azure Pipelines.",
```

small nit, adding a comma
```suggestion
  "description": "Sample project that demonstrates accessibility testing with TypeScript, Playwright, @axe-core/playwright, SARIF, and Azure Pipelines.",
```